src/frontend/src/components/ManufacturerDeleteButton.tsx

```diff
@@ -0,0 +1,80 @@
+import DeleteIcon from '@mui/icons-material/Delete';
+import { Box, IconButton } from '@mui/material';
+import { useState } from 'react';
+import ManufacturerDeleteButtonBlocker from '../components/ManufacturerDeleteButtonBlocker';
+import { useDeleteManufacturer } from '../hooks/bom.hooks';
+import { useToast } from '../hooks/toasts.hooks';
+
+interface ManufacturerDeleteButtonProps {
+  name: string;
+  onDelete: (name: string) => void;
+}
+
+const ManufacturerDeleteButton: React.FC<ManufacturerDeleteButtonProps> = ({
```

Comment:
Dont need to make this a whole component, try and combine this and the ManufactererDeleteButtonBlocker into one file. Also change the naming in manufactererDeleteModal, better naming. Take it out of the component folder and move it into the BOMConfig